helm/kubernetes-cluster-prep/templates/serviceaccount.yaml

```
@@ -2,13 +2,14 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ .Values.authnK8s.serviceAccount.name | default "authn-k8s-serviceaccount" }}
+  name: {{ .Values.authnK8s.serviceAccount.name | default "conjur-serviceaccount" }}
   labels:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
-    app.kubernetes.io/name: authn-k8s-serviceaccount
-    app.kubernetes.io/component: authn-k8s-identity
+    app.kubernetes.io/name: conjur-serviceaccount
+    app.kubernetes.io/component: conjur-identity
```

```suggestion
    app.kubernetes.io/component: conjur-cluster-identity
```
suggestion - since that's what this is, right?

---

Yes, agreed.